File: projects/dotnet/crud/LibraryManagement/Book.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LibraryManagement
{
    public class Book
    {
        public int BookId { get; set; }
        public string Name { get; set; }
        public DateTime issuedDate { get; set; }
        public Book(int BookId, string Name, DateTime issuedDate)
        {
            this.BookId = BookId;
            this.Name = Name;
            this.issuedDate = issuedDate;
        }
    }
}

File: LLDQuestions/library/library/src/Book.java
import java.time.LocalDate;

public class Book {
    String bookId;
    String name;
    String author;
    String publicationDate;
    Status status;
    LocalDate issuedDate;

    public Book(String bookId, String name, String author, String publicationDate, Status status, LocalDate issuedDate) {
        this.bookId = bookId;
        this.name = name;
        this.author = author;
        this.publicationDate = publicationDate;
        this.status = status;
        this.issuedDate = null;
    }
}

